### PR TITLE
Masterdiet 12

### PR DESCRIPTION
**Pull request by RafisSomeone**
[MASTERDIET-12] Slider added.

### CODE REVIEW

File: src/form/Start.css [hunk 1]
```diff
@@ -80,13 +80,28 @@ img {
...
+    float: left;
+    width: 33.33%;
+    padding: 5px;
+}
+
+.row::after {
+    content: "";
+    clear: both;
+    display: table;
 }
```
Comment (jakubowiczish) [inline src/form/Start.css:107]: Remember to always use ```standard --fix``` command

File: src/form/Start.js [hunk 2]
```diff
@@ -80,17 +90,36 @@ class StartForm extends Component {
...
+        const year = date.getFullYear();
+        let month = (1 + date.getMonth()).toString();
+        month = month.length > 1 ? month : '0' + month;
+        let day = date.getDate().toString();
+        day = day.length > 1 ? day : '0' + day;
+        return day + '/' + month + '/' + year;
+    }
 
     render() {
         const heightTest = /^[4-9][0-9]$|^1[0-9][0-9]$|^2[0-4][0-9]$|^250$/i;
```
Comment (jakubowiczish) [inline src/form/Start.js:125]: Maybe these checks can be separated into other methods?

Comment (RafisSomeone) [reply src/form/Start.js:125]: Method added.

File: src/form/Start.css (hunk 1 above)
Comment (RafisSomeone) [reply src/form/Start.css:107]: fixed

**Pull request merged**